### PR TITLE
Add [alt] attributes to images 

### LINKED ISSUE #9629: Image elements do not have [alt] attributes

### PR DESCRIPTION
**Pull request by Souvik-Cyclic**
Closes #9629 

Adds [alt] attributes to images for Improving Accessibility Score of Book Pages

### Stakeholders
@cdrini

### CODE REVIEW

File: openlibrary/macros/EditionNavBar.html [hunk 1]
```diff
@@ -33,6 +33,8 @@
   </ul>
   $if not desktop_only:
     <span class="nav-arrow">
-      <img src="/static/images/nav-arrow.svg" role="img">
+      <button title="$_('Go to next section')">
+        <img src="/static/images/nav-arrow.svg" alt="">
+      </button>
     </span>
```
Comment (cdrini) [inline openlibrary/macros/EditionNavBar.html:39]: Heads up: This was meant to replace the span with the button, and you'll have to update some the CSS as well

File: openlibrary/macros/EditionNavBar.html [hunk 2]
```diff
@@ -32,7 +32,7 @@
     </li>
   </ul>
   $if not desktop_only:
-    <span class="nav-arrow">
-      <img src="/static/images/nav-arrow.svg" role="img">
-    </span>
+  <button class="nav-arrow" title="$_('Go to next section')">
+    <img src="/static/images/nav-arrow.svg" alt="">
+  </button>
```
Comment (jimchamp) [inline openlibrary/macros/EditionNavBar.html:37]:
```suggestion
    <button class="nav-arrow" title="$_('Go to next section')">
      <img src="/static/images/nav-arrow.svg" alt="">
    </button>
```
Indentation level of this code block should remain the same.

Comment (jimchamp) [reply openlibrary/macros/EditionNavBar.html:37]: Fix styling of button:
image

**Review by jimchamp — CHANGES_REQUESTED** [review]
Thanks for your patience, @Souvik-Cyclic!

This is mostly good.  The styling of the `Go to next section` button will need to be updated before this can be merged in.

**Comment by Souvik-Cyclic** [comment]
I fixed the indentation, but I had trouble adjusting the button's styling. I tried changing the CSS, but the changes didn't take effect. @jimchamp, could you help me with this?

**Comment by jimchamp** [comment]
Sure!  Every time a change is made to a `.less` file, the change needs to be compiled into CSS before it can take effect.  You can do this by running the following:
```bash
docker compose exec web make css
```

More information about developing Open Library's front-end can be found [here](https://github.com/internetarchive/openlibrary/wiki/Frontend-Guide).

**Comment by Souvik-Cyclic** [comment]
I am closing this PR due to an error on my part. I will submit a new one shortly.

**Comment by Souvik-Cyclic** [comment]
@jimchamp What exactly happened that so many commits started showing in this one ?

**Pull request closed without merging**